synfig-studio/src/gui/resourcehelper.cpp

```diff
@@ -37,33 +37,29 @@
 
 synfig::String studio::ResourceHelper::get_image_path()
 {
-#ifdef _WIN32
-#	ifdef IMAGE_DIR
-#		undef IMAGE_DIR
-#		define IMAGE_DIR "share/pixmaps"
-#	endif
-#endif
-
 #ifndef IMAGE_DIR
 #	define IMAGE_DIR "/usr/local/share/pixmaps/synfigstudio"
 #endif
 
 	std::string imagepath;
-#ifdef _WIN32
-	imagepath=App::get_base_path()+'/'+IMAGE_DIR;
-#else
-	imagepath=IMAGE_DIR;
-#endif
 	char* synfig_root=getenv("SYNFIG_ROOT");
 	if(synfig_root) {
 		imagepath=synfig_root;
-		// Only class About didn't use the synfigstudio directory when using
+		//  Only class About didn't use the synfigstudio directory when using
 		//  SYNFIG_ROOT env variable. However, if it weren't set, it would use
 		//  IMAGE_DIR builtin variable that includes "synfigstudio". It means
 		//  that that About icon is in both folders. Therefore, it is safe to
 		//  choose this path.
 		imagepath+="/share/pixmaps/synfigstudio";
 	}
+	else {
+#if defined CMAKE_BUILD || _WIN32
+        imagepath=App::get_base_path()+"/share/pixmaps/synfigstudio";
```

Comment:
If the image path on Windows is `/share/pixmaps/`, then should I also make changes according to that path [here](https://github.com/synfig/synfig/pull/1247/files#r393446705) because I've moved everything to `PIXMAPS_DIR=${ROOT}/share/pixmaps/synfigstudio/` without taking into consideration the system it's built on.

Reply:
Good question! Since we don’t need the same behavior for autotools / cmake, I suggest using the correct path "/share/pixmaps/ synfigstudio" for CMake build and "/share/pixmaps" for autotools build.